src/MCPClient/debian/control

```diff
@@ -45,5 +45,6 @@ Depends:
 	tree,
 	ufraw,
 	unrar-free,
+	python-unidecode,
```

Comment:
Why is unidecode needed?  If there is a python dependency, couldn't this get added via the archivematica-common requirements file?  That way it wouldn't need to get added in multiple places (spec file for rpm, ansible vars file).